packages/xenctrl/xenctrl.0.9.29/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "1.2"
+maintainer: "dave@recoil.org"
+authors: [
+  "David Scott"
+  "Jonathan Ludlam"
+  "Andrew Cooper"
+  "Bob Ball"
+  "Euan Harris"
+  "John Else"
+  "Mike McClurg"
+  "Rob Hoes"
+  "Si Beaumont"
+  "Thomas Sanders"
+  "Vincent Bernadoff"
+]
+homepage: "https://github.com/xapi-project/ocaml-xen-lowlevel-libs"
+bug-reports: "https://github.com/xapi-project/ocaml-xen-lowlevel-libs/issues"
+license: "LGPL"
+dev-repo: "https://github.com/xapi-project/ocaml-xen-lowlevel-libs.git"
+tags: [
+  "org:mirage"
+  "org:xapi-project"
+]
+build: [
+  ["./configure"]
+  [make]
+]
+install: [
+  [make "install" "BINDIR=%{bin}%"]
+]
+remove: [
+  ["./configure"]
+  [make "uninstall"]
+]
+depends: [
+  "ocamlfind"
+  "lwt"
+  "cmdliner"
+]
+depexts: [
+  [["debian"] ["libxen-dev" "uuid-dev"]]
+  [["ubuntu"] ["libxen-dev" "uuid-dev"]]
+  [["centos"] ["xen-devel"]]
+  [["xenserver"] ["xen-dom0-libs-devel" "xen-libs-devel"]]
+]
+available: [ocaml-version: >= "4.00.0"]
```

Comment:
There's an extra colon here:
`available: [ocaml-version: >= "4.00.0"]` should be
`available: [ocaml-version >= "4.00.0"]`